CatAnalyzer/plugins/TtbarDiLeptonAnalyzer.cc

```diff
@@ -318,9 +310,11 @@ void TtbarDiLeptonAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSe
   if      ( b_channel == CH_ELEL ) iEvent.getByToken(trigTokenELEL_, trigHandle);
   else if ( b_channel == CH_MUMU ) iEvent.getByToken(trigTokenMUMU_, trigHandle);
   else if ( b_channel == CH_MUEL ) iEvent.getByToken(trigTokenMUEL_, trigHandle);
-  if ( !trigHandle.isValid() ) return;
+  if ( !trigHandle.isValid() ){
+    ttree_->Fill();
+	return;
+  }
   b_tri = *trigHandle;
-  //if (!tri) return;
```

Comment:
You can modify this part something like
```
if ( trigHandle.isValid() ) b_tri = *trigHandle;
```
to continue to next step even if the module fails to get the trigger bit info.